.rubocop_todo.yml

```diff
@@ -228,10 +235,11 @@ RSpec/ScatteredLet:
 # Offense count: 9
 RSpec/SubjectStub:
   Exclude:
-    - 'spec/models/index_queue_spec.rb'
-    - 'spec/search_builders/argo/date_field_queries_spec.rb'
-    - 'spec/search_builders/argo/profile_queries_spec.rb'
-    - 'spec/services/version_service_spec.rb'
+    - 'spec/**/*'
```

Comment:
I'd prefer that we don't manually adjust this file and just run `--auto-gen-config`

Reply:
Makes sense. I'll replace this one.